max.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import "fmt"
+
+func max(x []string) string {
+	var l, s, p int
+	l = len(x)
+	s = len(x[0])
```

Comment:
Please use more obvious names for variables. I cannot understand from the variable name `s` what is stored in it.

Reply:
Are sure that slice `x` has an element with `0` index? 
We should check the length of slice.